examples/python/README.md

```diff
@@ -0,0 +1,7 @@
+### Pre-requisite
+- have python installed , preferably `>=3.0`
+- install the requirements mentioned in the `requirement.txt`
```

Comment:
`pip3 install -r requirements.txt` right? We should explain that here

Reply:
Done.